FWCore/MessageService/src/ELstatistics.cc

```diff
@@ -248,16 +248,16 @@ namespace edm {
         //
         if (n == 0) {
           s << "\n";
-          s << " type     category        sev    module        "
-               "subroutine        count    total\n"
-            << " ---- -------------------- -- ---------------- "
-               "----------------  -----    -----\n";
+          s << " type     category        sev         module            "
+               "   subroutine          count    total\n"
+            << " ---- -------------------- -- --------------------- "
```

Comment:
I think this printout would need 4 more `-` and spaces for the "module" column, because the width was increased by 9 from 16 to 21.

Reply:
True. Sorry that was careless of me. Now fixed.